tests/Propel/Tests/Generator/Builder/Om/QueryBuilderTest.php

```diff
@@ -1312,7 +1312,7 @@ public function testUseRelationExistsQuery()
     public function testUseRelationNotExistsQuery()
     {
         $expected = BookQuery::create()
-        ->useExistsQuery('Author', null, null, ExistsCriterion::TYPE_NOT_EXISTS)
+        ->useExistsQuery('Author', null, null, ExistsQueryCriterion::TYPE_NOT_EXISTS)
```

Comment:
[Minor] IMO better not to change the test, to make sure that it still works with child class `ExistsCriterion` instead of `ExistsQueryCriterion`
Same story with https://github.com/propelorm/Propel2/pull/1920/files#diff-9dd69f4d7d65724ce303aaa7434eec44b30f936b56f94fb9afe3d1bd6df748e5

Reply:
Ok, I changed those back.
Thank you for the review! That was very helpful.

Reply:
Thanks for the PR, it very useful improvement!